src/core/AutomatableModel.cpp

```diff
@@ -275,9 +275,10 @@ float AutomatableModel::inverseScaledValue( float value ) const
 
 QString AutomatableModel::displayValue( const float val ) const
 {
+	const FloatModel *floatmodel = dynamic_cast<const FloatModel*>( this );
 	switch( m_dataType )
 	{
-		case Float: return QString::number( castValue<float>( scaledValue( val ) ) );
+		case Float: return QString::number( castValue<float>( scaledValue( floatmodel->getRoundedValue() ) ) );
```

Comment:
This function is used to convert any value into a string, not just the current value. `Reset` and `Paste value` are now incorrect:
![image](https://cloud.githubusercontent.com/assets/772336/22788637/22e1b56c-eee9-11e6-8b83-2a1bfc197980.png)